packages/cli/src/platform/_lib/utils.ts

```diff
@@ -94,3 +94,43 @@ export const poll = async <F extends () => Promise<R>, R>(
 
   return result
 }
+
+export const printPpgInitOutput = ({
+  databaseUrl,
+  workspaceId,
+  projectId,
+  environmentId,
+}: {
+  databaseUrl: string
+  workspaceId: string
+  projectId: string
+  environmentId: string
+}) => `
+Database URL:
+${green(databaseUrl)}
+
+Project link:
+${link(`https://console.prisma.io/${workspaceId}/${projectId}/${environmentId}/dashboard`)}
+
+${bold('Next steps')}
+${bold('1. Define your database schema')}
+Open the ${green('schema.prisma')} file and define your first models. Go to ${link(
+  underline('pris.ly/ppg-quickstart'),
```

Comment:
Do you want to set the protocol on this for consistency? (Or does `link()` do it automatically 😅)

Reply:
Let's add it!